### PR TITLE
document for add comment

### PR DESCRIPTION
**Pull request by mvn-caonguyen-dn**
## Description
A few sentences describing the overall goals of the pull request's commits.
## Reference Requirements
- Product Backlog ID: PBI-40
- Task ID: #40.4

## Reference Isuees
#{issue_id}

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
check cmt nha

File: docs/api-references/comment.md [hunk 1]
```diff
@@ -117,3 +117,84 @@ Comments for Book
...
+| parent_id | Number | Parent comment id |
+| created_at | String | Create comment time |
+| updated_at | String | Update comment time |
+| deleted_at | String | Delete comment time |
+
+```json
+{
+    "meta": {
+        "status": "successfully",
+        "code": 200
```
Comment (at-nhannguyen2) [inline docs/api-references/comment.md]: create là code 201

File: docs/api-references/comment.md [hunk 2]
```diff
@@ -117,3 +117,84 @@ Comments for Book
...
+        "commentable_id": 1,
+        "commentable_type": "book",
+        "name": "Greta Lehner",
+        "team": "SA",
+        "avatar_url": "http://flappybook.tech/tmp/7c5266db2cd916e1af404a688980e4dd.jpg",
+        "is_admin": 0,
+        "parent_id": null,
+        "created_at": "2018-02-08 16:34:10",
+        "updated_at": "2018-02-08 16:34:10",
+        "deleted_at": null
```
Comment (at-nhannguyen2) [inline docs/api-references/comment.md]: cái này có cần thiết lấy ra k

**Review by ghost — COMMENTED** [review]
check cmt

File: docs/api-references/comment.md [hunk 3]
```diff
@@ -117,3 +117,88 @@ Comments for Book
...
+| code | Number | HTTP status codes |
+| error | Object | Object error |
+| message | Object | Object message |
+| comment | Array | Error comment |
+
+```json
+{
+    "meta": {
+        "status": "Failed",
+        "code": 422
```
Comment (at-huydq) [inline docs/api-references/comment.md]: trả về 
status : `Bad Request`
code :  `400`

**Comment by at-huydq** [comment]
LGTM

**Pull request merged**